src/cmd/delegation_backend/main_bpu.go

```diff
@@ -14,6 +16,23 @@ import (
 	sheets "google.golang.org/api/sheets/v4"
 )
 
+var requestsPerPkHourly int
+
+func init() {
+	var defaultValue = 120
+	var err error
+
+	envVarValue, exists := os.LookupEnv("REQUESTS_PER_PK_HOURLY")
+	if exists {
+		requestsPerPkHourly, err = strconv.Atoi(envVarValue)
+		if err != nil {
+			requestsPerPkHourly = defaultValue
```

Comment:
Maybe add a warning log so that is clear to the user that the value set was not valid (it might have been a typo the user didn't notice).